config/routes.rb

```diff
@@ -1,6 +1,16 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "users#index"
 
+  resources :users do
+    resources :posts
+  end
+
+  # get "/users", to: "users#index"
+  # get "/users/:id", to: "users#show"
+  # get "/users/:id/:posts", to: "posts#index"
+  # get "users/:id/:posts/:id", to: "posts#show" 
```

Comment:
- [ ] Please remove the unused code to ensure a clean and optimized code base. 🙏